### PR TITLE
Passive port resolvers

### PR DESCRIPTION
**Pull request by DevAndyroo**
Passive port resolvers

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -17,13 +17,13 @@
 	<parent>
 		<artifactId>ftpserver-parent</artifactId>
 		<groupId>org.apache.ftpserver</groupId>
-		<version>1.0.6-osgi</version>
+		<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline core/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: core/pom.xml [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
 	</parent>
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.apache.ftpserver</groupId>
 	<artifactId>ftpserver-core</artifactId>
 	<name>Apache FtpServer Core</name>
-	<version>1.0.6-osgi</version>
+	<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline core/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: core/src/main/java/org/apache/ftpserver/DataConnectionConfigurationFactory.java [hunk 3]
```diff
@@ -49,7 +50,8 @@
     
     private String passiveAddress;
     private String passiveExternalAddress;
-    private PassivePorts passivePorts = new PassivePorts(Collections.<Integer>emptySet(), true);
+    //private PassivePorts passivePorts = new PassivePorts(Collections.<Integer>emptySet(), true);
```
Comment (nsk2000) [inline core/src/main/java/org/apache/ftpserver/DataConnectionConfigurationFactory.java]: Remove commented out code

File: core/src/main/java/org/apache/ftpserver/impl/DefaultDataConnectionConfiguration.java [hunk 4]
```diff
@@ -44,8 +44,9 @@
     
     private String passiveAddress;
     private String passiveExternalAddress;
-    private PassivePorts passivePorts;
-    
+    //private PassivePorts passivePorts;
```
Comment (nsk2000) [inline core/src/main/java/org/apache/ftpserver/impl/DefaultDataConnectionConfiguration.java:47]: Remove commented out code

**Comment by ahaigh** [comment]
R2

File: distribution/pom.xml [hunk 5]
```diff
@@ -22,14 +22,14 @@
   <parent>
     <artifactId>ftpserver-parent</artifactId>
     <groupId>org.apache.ftpserver</groupId>
-    <version>1.0.6-osgi</version>
+    <version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline distribution/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

**Review by ahaigh — APPROVED** [review]
R2 - LGTM

File: distribution/pom.xml [hunk 6]
```diff
@@ -22,14 +22,14 @@
...
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.ftpserver</groupId>
   <artifactId>ftpserver</artifactId>
   <packaging>pom</packaging>
   <name>Apache FtpServer</name>
-  <version>1.0.6-osgi</version>
+  <version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline distribution/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/ftpserver-example-spring-war/pom.xml [hunk 7]
```diff
@@ -18,7 +18,7 @@
     <parent>
         <artifactId>ftpserver-parent</artifactId>
         <groupId>org.apache.ftpserver</groupId>
-        <version>1.0.6-osgi</version>
+        <version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/ftpserver-example-spring-war/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/ftpserver-example-spring-war/pom.xml [hunk 8]
```diff
@@ -27,7 +27,7 @@
 	<artifactId>ftpserver-spring-war</artifactId>
 	<name>FtpServer Spring web project example</name>
 	<packaging>war</packaging>
-	<version>1.0.6-osgi</version>
+	<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/ftpserver-example-spring-war/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/ftpserver-osgi-ftplet-service/pom.xml [hunk 9]
```diff
@@ -19,15 +19,15 @@
 	<parent>
 		<artifactId>ftpserver-parent</artifactId>
 		<groupId>org.apache.ftpserver</groupId>
-		<version>1.0.6-osgi</version>
+		<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/ftpserver-osgi-ftplet-service/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/ftpserver-osgi-ftplet-service/pom.xml [hunk 10]
```diff
@@ -19,15 +19,15 @@
...
 		<relativePath>../../pom.xml</relativePath>
 	</parent>
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.apache.ftpserver.examples</groupId>
 	<artifactId>ftpserver-osgi-ftplet-service</artifactId>
 	<name>FtpServer OSGi Ftplet service example</name>
 	<packaging>bundle</packaging>
-	<version>1.0.6-osgi</version>
+	<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/ftpserver-osgi-ftplet-service/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/ftpserver-osgi-spring-service/pom.xml [hunk 11]
```diff
@@ -19,15 +19,15 @@
 	<parent>
 		<artifactId>ftpserver-parent</artifactId>
 		<groupId>org.apache.ftpserver</groupId>
-		<version>1.0.6-osgi</version>
+		<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/ftpserver-osgi-spring-service/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/ftpserver-osgi-spring-service/pom.xml [hunk 12]
```diff
@@ -19,15 +19,15 @@
...
 		<relativePath>../../pom.xml</relativePath>
 	</parent>
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.apache.ftpserver.examples</groupId>
 	<artifactId>ftpserver-osgi-spring-service</artifactId>
 	<name>FtpServer OSGi Spring-DM example</name>
 	<packaging>bundle</packaging>
-	<version>1.0.6-osgi</version>
+	<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/ftpserver-osgi-spring-service/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/pom.xml [hunk 13]
```diff
@@ -21,14 +21,14 @@
   <parent>
       <artifactId>ftpserver-parent</artifactId>
       <groupId>org.apache.ftpserver</groupId>
-      <version>1.0.6-osgi</version>
+      <version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/pom.xml [hunk 14]
```diff
@@ -21,14 +21,14 @@
...
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.ftpserver</groupId>
   <artifactId>ftpserver-examples</artifactId>
   <packaging>pom</packaging>
   <name>Apache FtpServer Examples</name>
-  <version>1.0.6-osgi</version>
```
Comment (nsk2000) [inline examples/pom.xml:31]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: examples/pom.xml [hunk 15]
```diff
@@ -21,14 +21,14 @@
...
+  <version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline examples/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: ftplet-api/pom.xml [hunk 16]
```diff
@@ -17,12 +17,12 @@
 	<parent>
 		<artifactId>ftpserver-parent</artifactId>
 		<groupId>org.apache.ftpserver</groupId>
-		<version>1.0.6-osgi</version>
+		<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline ftplet-api/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: ftplet-api/pom.xml [hunk 17]
```diff
@@ -17,12 +17,12 @@
...
 	</parent>
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.apache.ftpserver</groupId>
 	<artifactId>ftplet-api</artifactId>
-	<version>1.0.6-osgi</version>
+	<version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline ftplet-api/pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

File: pom.xml [hunk 18]
```diff
@@ -28,7 +28,7 @@
   <artifactId>ftpserver-parent</artifactId>
   <packaging>pom</packaging>
   <name>Apache FtpServer Parent</name>
-  <version>1.0.6-osgi</version>
+  <version>1.0.6-atos1-SNAPSHOT</version>
```
Comment (nsk2000) [inline pom.xml]: Perhaps 1.0.6-osgi-1.0.0 would be better

**Comment by nsk2000** [comment]
R1

**Pull request merged**